src/routes/Accounts/AccountTabs/Overview.tsx

```diff
@@ -21,62 +17,137 @@ import SearchSortField from 'Components/Search&Sort'
 import useTransactions from 'Hooks/transactions/useTransactions'
 import { Account } from 'Data/types/Account'
 import { useNavigate } from 'react-router-dom'
-import EmptyOverviewImage from 'Svgx/EmptyOverviewImage'
 import ROUTES from 'Routes/data'
 import SortType from 'Types/SortType'
 import { useDataSync } from 'Providers/DataSyncProvider'
 import TransactionStatus from 'Components/TransactionStatus'
 import { accountGradientByOrder } from 'Utils/accountGradientByOrder'
+import EmptyOverview from 'Components/EmptyOverview'
 
-interface AccountOverviewProps {
-  account: Account
-  order: number
+interface SearchTransactionsProps {
+  address: string
 }
 
-const EmptyOverview = () => {
-  const $fontColor = useColorModeValue(
-    NAMED_COLORS.GREY,
-    NAMED_COLORS.PALE_GREY
+const SearchTransactions: FC<SearchTransactionsProps> = ({ address }) => {
+  const [$searchTerm, $setSearchTerm] = useState('')
+  const [$sortOrder, $setSortOrder] = useState<SortType>(SortType.ASC)
+  const [{ data: transactions = undefined, loaded }] = useTransactions(
+    address,
+    $searchTerm,
+    $sortOrder
   )
-  return (
-    <Flex mt="2rem" justifyContent="center">
-      <VStack w="25rem">
-        <chakra.h3 mb="1rem">You don’t have any transactions</chakra.h3>
-        <chakra.h5 mb="2.5rem !important" textAlign="center" color={$fontColor}>
-          When your account compiles transactions they will be listed here. To
-          produce a transactions, either{' '}
-          <Link
-            as={RouterLink}
-            to={ROUTES.SEND}
-            color={NAMED_COLORS.LIGHT_BLUE}
-          >
-            send
-          </Link>{' '}
-          or{' '}
-          <Link
-            as={RouterLink}
-            to={ROUTES.RECEIVE}
-            color={NAMED_COLORS.LIGHT_BLUE}
-          >
-            receive
-          </Link>{' '}
-          $IRON.
-        </chakra.h5>
-        <EmptyOverviewImage />
-      </VStack>
-    </Flex>
+
+  return loaded && transactions.length === 0 && !$searchTerm ? null : (
+    <>
+      <Box>
+        <chakra.h3 pb="1rem">Transactions</chakra.h3>
+        <SearchSortField
+          SearchProps={{
+            value: $searchTerm,
+            onChange: e => $setSearchTerm(e.target.value),
+          }}
+          SortSelectProps={{
+            onSelectOption: ({ value }) => $setSortOrder(value),
+          }}
+        />
+      </Box>
+      {transactions?.length === 0 ? (
+        <EmptyOverview
+          header="0 Results"
+          description="There aren’t any transactions with details that match your search input. "
+        />
+      ) : (
```

Comment:
As I right understand in interval of time, when we loading transaction we will see EmptyOverview? Is it ok?

Reply:
When address is not undefined we will have previously loaded transactions, so it will not be visible, at least I can't see it.